src/main/java/dk/kb/datahandler/oai/OaiResponseFiltering.java

```diff
@@ -66,6 +66,9 @@ public static void setRecordType(DsRecordDto dsRecord, String storageId) {
         } else if (storageId.contains("oai:man")) {
             dsRecord.setRecordType(RecordTypeDto.MANIFESTATION);
         }
+        else {
+            dsRecord.setRecordType(RecordTypeDto.DELIVERABLEUNIT);
+        }
```

Comment:
This is a bit hacky, I see why you are doing it though. Maybe add a small comment on why its done this way? - to give resources from the image collections a correct type.

Reply:
added more javadoc to method